src/RwaUrn.sol

```diff
@@ -190,7 +189,7 @@ contract RwaUrn2 {
         DaiAbstract(DaiJoinAbstract(daiJoin_).dai()).approve(daiJoin_, type(uint256).max);
         VatAbstract(vat_).hope(daiJoin_);
 
-        emit Recap(gemCap_);
+        emit File("gemCap", gemCap_);
         emit Rely(msg.sender);
         emit File("outputConduit", outputConduit_);
         emit File("jug", jug_);
```

Comment:
I'd put the `File("gemCap", gemCap_)` last to respect the order they are passed in.

Reply:
👍

Reply:
Fixed